config/routes.rb

```diff
@@ -63,6 +63,7 @@
         resources :campaigns, only: [:create]
         resources :communities, only: [:create]
         resources :mascots, only: [:create]
+        resources :mandatories, only: [:create]
```

Comment:
really small nitpick 🤣 `mandatories` comes before `mascots`

Reply:
Still working on the alphabet, getting there.